modules/openy_features/openy_menu/openy_menu.install

```diff
@@ -15,9 +15,17 @@ function openy_menu_update_8001() {
   $config .= 'tour.tour.openy-menu-overview.yml';
   $config_importer->update($config, 'tour.tour.openy-menu-overview', 'tips');
 }
+
 /**
  * Update menu tutorials.
  */
 function openy_menu_update_8002() {
   openy_menu_update_8001();
 }
+
+/**
+ * Update menu tutorials.
+ */
+function openy_menu_update_8003() {
+	openy_menu_update_8001();
```

Comment:
Update number two has already been completed and I do not need to change it then, right?
As I understood for update number three do 
function openy_menu_update_8003() {
	openy_menu_update_8002();
}
or you mean copy code from first update?